graspologic/match/gmp.py

```diff
@@ -32,17 +32,27 @@ class GraphMatch(BaseEstimator):
         the FAQ algorithm will undergo.
 
     init : string (default = 'barycenter') or 2d-array
-        The initial position chosen
-
-        If 2d-array, `init` must be :math:`m' x m'`, where :math:`m' = n - m`,
-        and it must be doubly stochastic: each of its rows and columns must
-        sum to 1.
+        The initial position chosen.
 
         "barycenter" : the non-informative “flat doubly stochastic matrix,”
-        :math:`J=1 \\times 1^T /n` , i.e the barycenter of the feasible region
+        :math:`J=1 \\times 1^T /n` , i.e the barycenter of the feasible region. This can
+        be thought of as the doubly stochastic matrix from which any permutation is 
+        equally likely.
 
         "rand" : some random point near :math:`J, (J+K)/2`, where K is some random
-        doubly stochastic matrix
+        doubly stochastic matrix.
+
+        For a custom initialization (a 1d- or 2d-array), the input repesents a 
+        permutation or assignment of the nonseeded vertices of `B`, and thus the 
+        dimension should be equal to the number of nonseeded vertices of `B`, :math:`m'`.
+        
+        If 2d-array, `init` must be :math:`m' x m'`, and it must be doubly stochastic: 
+        each of its rows and columns must sum to 1.
+
+        If 1d-array, `init` must be a set of permutation indices: the array should be 
+        of length :math:`m'` and the elements of the array must be integers from 
+        :math:`0 ... m' - 1`. The FAQ algorithm will start from the permutation matrix
```

Comment:
might be better to keep the vertex labels consistent? If they're passing in seeds and `init` indices?

Reply:
we can probably recover how to order this when building the perm matrix with argsort or something

Reply:
think `np.arange(n_unseed)[np.argsort(init)]` should do the trick

Reply:
i dont think i follow your suggestion

Reply:
or maybe i do - you're saying the permutation indices could be with respect to B instead of B22?

Reply:
yes

Reply:
think it might confuse the user if there's multiple vertex sets